buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -166,12 +183,20 @@ private void waitForUri(String description, String uri) {
         });
     }
 
+    /**
+     * Returns a stream of lines in the generated logs similar to Files.lines
+     *
+     * @return stream of log lines
+     */
+    public Stream<String> logLines() throws IOException {
```

Comment:
Is this intended for users of the plugin to do something with the log output in their build scripts?

Reply:
Yes, right now it's used by a test to verify the plugin is indeed loaded.